backend/src/polling_station/structs.rs

```diff
@@ -52,6 +52,7 @@ pub struct PollingStationStatusEntry {
 pub enum PollingStationStatus {
     FirstEntry,
     FirstEntryInProgress,
+    FirstEntryUnfinished,
```

Comment:
If you start data entry of a polling station, it has the "first entry unfinished" state while you're on the Recounted page. As soon as you submit and go to the Voters and Votes page, the state is updated the "first entry in progress".

Reply:
Ah let me check if I can set the default of continue to `true` somewhere

Reply:
Done in 56f56f04a185522c619255adb566a6e3bd92ee24 now it works as one would expect it